src/tree.c

```diff
@@ -27,7 +27,7 @@
 
 static const imask_t has_map[T_LAST_TREE_KIND] = {
    // T_ENTITY
-   (I_IDENT | I_PORTS | I_GENERICS | I_CONTEXT | I_DECLS | I_STMTS),
+   (I_IDENT | I_PORTS | I_GENERICS | I_CONTEXT | I_DECLS | I_STMTS | I_IVAL),
```

Comment:
As above, don't need this.

Reply:
Removed now, as discussed, _PX from X within architecture is counted from referenced entity.